meta-resin-common/recipes-core/systemd/systemd_%.bbappend

```diff
@@ -23,10 +23,10 @@ do_install_append() {
         find ${D} -name "getty@*.service" -delete
     fi
 
-    install -d -m 0755 /srv
+    install -d -m 0755 ${D}/srv
 
     ln -s ${datadir}/zoneinfo ${D}${sysconfdir}/localtime
-    ln -s /proc/self/mounts ${D}${sysconfdir}/mtab
+    ln -s ../proc/self/mounts ${D}${sysconfdir}/mtab
```

Comment:
I already have this in a beaglebone:

root@beaglebone:~# ls -al /etc/mtab                                                                                                                                                                                                          
lrwxrwxrwx 1 root root 19 Jan  1  2000 /etc/mtab -> ../proc/self/mounts

Hence, I think you need to investigate and see who sets this. Because it looks like what you do here is not taken into account